docs/cli.md

```diff
@@ -0,0 +1,309 @@
+  project: perses
+spec:
+  default: false
+  plugin:
+    kind: PrometheusDatasource
+    spec:
+      proxy:
+        kind: HTTPProxy
+        spec:
+          allowed_endpoints:
+          - endpoint_pattern: /api/v1/labels
+            method: POST
+          - endpoint_pattern: /api/v1/series
+            method: POST
+          - endpoint_pattern: /api/v1/metadata
+            method: GET
+          - endpoint_pattern: /api/v1/query
+            method: POST
+          - endpoint_pattern: /api/v1/query_range
+            method: POST
+          - endpoint_pattern: /api/v1/label/([a-zA-Z0-9_-]+)/values
+            method: GET
+          url: https://prometheus.demo.do.prometheus.io
+```
+
+Or in JSON:
+
+```bash
+$ percli describe dts PrometheusDemo -ojson | jq
+
+{
+  "kind": "Datasource",
+  "metadata": {
+    "name": "PrometheusDemo",
+    "created_at": "0001-01-01T00:00:00Z",
+    "updated_at": "0001-01-01T00:00:00Z",
+    "project": "perses"
+  },
+  "spec": {
+    "default": false,
+    "plugin": {
+      "kind": "PrometheusDatasource",
+      "spec": {
+        "proxy": {
+          "kind": "HTTPProxy",
+          "spec": {
+            "allowed_endpoints": [
+              {
+                "endpoint_pattern": "/api/v1/labels",
+                "method": "POST"
+              },
+              {
+                "endpoint_pattern": "/api/v1/series",
+                "method": "POST"
+              },
+              {
+                "endpoint_pattern": "/api/v1/metadata",
+                "method": "GET"
+              },
+              {
+                "endpoint_pattern": "/api/v1/query",
+                "method": "POST"
+              },
+              {
+                "endpoint_pattern": "/api/v1/query_range",
+                "method": "POST"
+              },
+              {
+                "endpoint_pattern": "/api/v1/label/([a-zA-Z0-9_-]+)/values",
+                "method": "GET"
+              }
+            ],
+            "url": "https://prometheus.demo.do.prometheus.io"
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+### Delete data
+
+To remove a resource, you can use the `delete` command :
+
+```bash
+$ percli delete dashboard Demo
+
+Dashboard Demo has been deleted
+```
+
+## Advanced Commands
+
+### Linter
+
+The CLI provides a command `lint` that is able to validate any data supported by Perses.
+
+Note that it doesn't necessary mean you won't face any issues when applying them.
+
+```bash
+$ percli lint -f ./resource.json
+```
+
+By default, the command doesn't require any remote server. We are providing a flag `--online` that will tell the CLI to
+use a remote Perses server for additional validation. For example, when it will have to validate a dashboard, it will
+use the endpoint `/api/validate/dashboards`. That can be useful if you want to be sure that your dashboard is compatible
+with the server (because it will use the remote CUE schemas instead of the local one)
```

Comment:
I'd say sth more high level like "because it will match the plugins known to the server instead of the local ones"

Reply:
ah yes good idea. Thanks !